jobs/delorean/jenkinsfiles/1.0/cluster/openshift/integreatly/test/Jenkinsfile

```diff
@@ -126,11 +126,11 @@ pipeline {
             [$class: 'StringParameterValue', name: 'CUSTOMER_ADMIN_PASSWORD', value: 'Password1'],
             [$class: 'StringParameterValue', name: 'TIMEOUT_THRESHOLD', value: '2'],
             [$class: 'StringParameterValue', name: 'NAMESPACE_PREFIX', value: 'openshift-'],
-            [$class: 'StringParameterValue', name: 'WEBAPP_URL', value: "https://tutorial-web-app-openshift-webapp.apps.${testOptions.clusterName}.skunkhenry.com"],
+            [$class: 'StringParameterValue', name: 'WEBAPP_URL', value: "https://tutorial-web-app-openshift-webapp.apps.${testOptions.clusterName}.tower.cloudservices.rhmw.io"],
             [$class: 'BooleanParameterValue', name: 'CLEAN_RESOURCES', value: true],
             [$class: 'StringParameterValue', name: 'MANIFEST_VERSION', value: 'master'],
             [$class: 'BooleanParameterValue', name: 'TESTING_MASTER', value: false],
-            [$class: 'StringParameterValue', name: 'SSO_URL', value: "https://sso-openshift-sso.apps.${testOptions.clusterName}.skunkhenry.com"],
+            [$class: 'StringParameterValue', name: 'SSO_URL', value: "https://sso-openshift-sso.apps.${testOptions.clusterName}.tower.cloudservices.rhmw.io"],
```

Comment:
could `testOptions.domainName` be used here instead of hardcoded value (`tower.cloudservices.rhmw.io`)?